src/utils/features.ts

```diff
@@ -0,0 +1,37 @@
+import { TFeatureFlags, TThemeConfig } from "@/types";
+
+interface IArgs {
+  configFeature?: boolean;
+  featureFlag?: boolean;
+}
+
+// Determines if an app feature is on or off by combining app-level config and PostHog feature flags
+export const isFeatureEnabled = ({ configFeature, featureFlag }: IArgs): boolean => {
+  if (configFeature === undefined && featureFlag === undefined) return false; // Shouldn't be using this fn
```

Comment:
As mentioned in call - I think here we maybe default to true? Because if it isn't in the config and there isn't a feature flag for it, makes me think its just a 'normal' feature?